src/uncrustify.cpp

```diff
@@ -1024,12 +1026,12 @@ int main(int argc, char *argv[])
 
       // Issue #3427
       init_keywords_for_language();
-      uncrustify_file(fm, stdout, parsed_file, dump_file_name);
+      uncrustify_file(fm, stdout, parsed_file, dump_file_name, false);
```

Comment:
TODO: should this be is_quiet?

Reply:
If the question is about default behavior, I think it should be "not quiet" since we have a command option to request to be quiet, but not viceversa.
If the question is about whether `false` is right or wrong, then I need to study the code further.

Reply:
I meant it as a TODO note to myself.